### ISSUE TITLE
use case for ScopeInterface missing in PinTerminals source config

### ISSUE THREAD

**Issue by jv-etailors** [paynl/magento2-plugin#165]
When changing the scope in the Magento configuration, the system errors, being unable to adjust the configuration for that scope.

**Preconditions:**
paynl/magento2-plugin: 2.5.0
magento2/product-community-edition: 2.4.x

**Steps to reproduce**
- Go to Stores > Configuration, Sales > Payment Methods
- Make sure the tokens are set correctly
- Change the scope of the configuration to a website or store view

**Expected Result**
Configuration for the scope is show

**Actual Result**
An error occurs: 
```
main.CRITICAL: Error: Class 'Paynl\Payment\Model\Config\Source\ScopeInterface' not found in /app/vendor/paynl/magento2-plugin/Model/Config/Source/PinTerminals.php:133
Stack trace:
#0 /app/vendor/paynl/magento2-plugin/Model/Config/Source/PinTerminals.php(80): Paynl\Payment\Model\Config\Source\PinTerminals->getConfigValue('payment/paynl_p...')
#1 /app/vendor/paynl/magento2-plugin/Model/Config/Source/PinTerminals.php(58): Paynl\Payment\Model\Config\Source\PinTerminals->toArray()
#2 /app/vendor/magento/module-config/Model/Config/Structure/Element/Field.php(461): Paynl\Payment\Model\Config\Source\PinTerminals->toOptionArray(false)
#3 /app/vendor/magento/module-config/Model/Config/Structure/Element/Field.php(379): Magento\Config\Model\Config\Structure\Element\Field->_getOptionsFromSourceModel(Object(Paynl\Payment\Model\Config\Source\PinTerminals))
#4 /app/vendor/magento/module-config/Block/System/Config/Form.php(394): Magento\Config\Model\Config\Structure\Element\Field->getOptions()
#5 /app/vendor/magento/module-config/Block/System/Config/Form.php(316): Magento\Config\Block\System\Config\Form->_initElement(Object(Magento\Config\Model\Config\Structure\Element\Field\Interceptor), Object(Magento\Framework\Data\Form\Element\Fieldset), 'payment/paynl_p...', '', '')
```

Attached is a patch file that resolves the issue
[paynl_pin_terminals.txt](https://github.com/paynl/magento2-plugin/files/7702889/paynl_pin_terminals.txt)

**Comment by woutse** [comment]
Thanks for your message. We will update this asap.

**Comment by woutse** [comment]
This is fixed in 2.5.1 Thanks for reporting!
